crates/sui-framework/sources/package.move

```diff
@@ -28,9 +28,9 @@ module sui::package {
     const COMPATIBLE: u8 = 0;
     /// Add new functions or types, or change dependencies, existing
     /// functions can't change.
-    const ADDITIVE: u8 = 1;
+    const ADDITIVE: u8 = 128;
```

Comment:
I also "spaced out" the constants for upgrade policies, in case we need to add more after the fact, but without losing the nice property that higher numbers are more restrictive.